lib/Swagger/src/Discussion/Models/LocalDateTime.php

```diff
@@ -0,0 +1,581 @@
+ * @license     http://www.apache.org/licenses/LICENSE-2.0 Apache Licene v2
+ * @link        https://github.com/swagger-api/swagger-codegen
+ */
+class LocalDateTime implements ArrayAccess
+{
+    /**
+      * The original name of the model.
+      * @var string
+      */
+    protected static $swaggerModelName = 'LocalDateTime';
+
+    /**
+      * Array of property to type mappings. Used for (de)serialization
+      * @var string[]
+      */
+    protected static $swaggerTypes = array(
+        'hour' => 'int',
+        'minute' => 'int',
+        'second' => 'int',
+        'nano' => 'int',
+        'day_of_year' => 'int',
+        'day_of_week' => 'string',
+        'month' => 'string',
+        'day_of_month' => 'int',
+        'year' => 'int',
+        'month_value' => 'int',
+        'chronology' => '\Swagger\Client\Discussion\Models\Chronology'
+    );
+
+    public static function swaggerTypes()
+    {
+        return self::$swaggerTypes;
+    }
+
+    /**
+     * Array of attributes where the key is the local name, and the value is the original name
+     * @var string[]
+     */
+    protected static $attributeMap = array(
+        'hour' => 'hour',
+        'minute' => 'minute',
+        'second' => 'second',
+        'nano' => 'nano',
+        'day_of_year' => 'dayOfYear',
+        'day_of_week' => 'dayOfWeek',
+        'month' => 'month',
+        'day_of_month' => 'dayOfMonth',
+        'year' => 'year',
+        'month_value' => 'monthValue',
+        'chronology' => 'chronology'
+    );
+
+    public static function attributeMap()
+    {
+        return self::$attributeMap;
+    }
+
+    /**
+     * Array of attributes to setter functions (for deserialization of responses)
+     * @var string[]
+     */
+    protected static $setters = array(
+        'hour' => 'setHour',
+        'minute' => 'setMinute',
+        'second' => 'setSecond',
+        'nano' => 'setNano',
+        'day_of_year' => 'setDayOfYear',
+        'day_of_week' => 'setDayOfWeek',
+        'month' => 'setMonth',
+        'day_of_month' => 'setDayOfMonth',
+        'year' => 'setYear',
+        'month_value' => 'setMonthValue',
+        'chronology' => 'setChronology'
+    );
+
+    public static function setters()
+    {
+        return self::$setters;
+    }
+
+    /**
+     * Array of attributes to getter functions (for serialization of requests)
+     * @var string[]
+     */
+    protected static $getters = array(
+        'hour' => 'getHour',
+        'minute' => 'getMinute',
+        'second' => 'getSecond',
+        'nano' => 'getNano',
+        'day_of_year' => 'getDayOfYear',
+        'day_of_week' => 'getDayOfWeek',
+        'month' => 'getMonth',
+        'day_of_month' => 'getDayOfMonth',
+        'year' => 'getYear',
+        'month_value' => 'getMonthValue',
+        'chronology' => 'getChronology'
+    );
+
+    public static function getters()
+    {
+        return self::$getters;
+    }
+
+    const DAY_OF_WEEK_MONDAY = 'MONDAY';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "MONDAY" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)